packages/harmony/src/components/button/FilterButton/types.ts

```diff
@@ -77,7 +92,7 @@ export type FilterButtonProps = {
   /**
    * What to do when the value is changed
    */
-  onChange?: (value: string) => void
+  onChange?: (value: Value) => void
```

Comment:
Did we not want to turn this into a change event?

Reply:
oh yeah duh :)